src/DeviceRegistry/DeviceRegistry.Autorest/docs/Get-AzDeviceRegistryOperationStatus.md

```diff
@@ -0,0 +1,123 @@
+---
+external help file:
+Module Name: Az.DeviceRegistry
+online version: https://learn.microsoft.com/powershell/module/az.deviceregistry/get-azdeviceregistryoperationstatus
+schema: 2.0.0
+---
+
+# Get-AzDeviceRegistryOperationStatus
+
+## SYNOPSIS
+Returns the current status of an async operation.
+
+## SYNTAX
+
+```
+Get-AzDeviceRegistryOperationStatus -Location <String> -OperationId <String> [-SubscriptionId <String[]>]
```

Comment:
Do we need this? Current pwsh commandlets for Create/Update resources automatically wait for async operation status to complete, as well as this pwsh command cannot get the async op uri signing params (see [here](https://armwiki.azurewebsites.net/api_contracts/AsyncOperationSigningAndValidation.html#async-operation-uri-signing) to successfully run